content/docs/intro/cloud-providers/fastly/_index.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Fastly
+meta_desc: This page provides an overview of the Fastly Provider for Pulumi.
+menu:
+  intro:
+    parent: cloud-providers
+    identifier: clouds-fastly
+    weight: 5
+
+aliases: ["/docs/reference/clouds/fastly/"]
+---
+
+<img src="/logos/tech/fastly.svg" align="right" class="h-16 px-8 pb-4">
+
+The Fastly provider for Pulumi can be used to provision any of the cloud resources available in [Fastly](https://www.fastly.com/).
+The Fastly provider must be configured with credentials to deploy and update resources in Fastly.
+
+See the [full API documentation]({{< relref "/docs/reference/pkg/nodejs/pulumi/fastly" >}}) for complete details of the available Fastly provider APIs.
+
+## Setup
+
+The Fastly provider supports several options for providing access to Fastly credentials.  See the [Fastly setup page]({{< relref "setup" >}}) for details.
+
+## Example
+
+{{< langchoose csharp >}}
+
+```javascript
+const fastly = require("@pulumi/fastly")
+
+const service = new fastly.Servicev1("my-service", {
+  backends: [{
+       address: "127.0.0.1",
+       name: "localhost",
+       port: 80
+   }],
+  domains: [{
+       comment: "demo",
+       name: "demo.pulumi.com",
+  }],
+  forceDestroy: true,
+});
+```
+
+```typescript
+import * as fastly from "@pulumi/fastly";
+
+const service = new fastly.Servicev1("my-service", {
+  backends: [{
+       address: "127.0.0.1",
+       name: "localhost",
+       port: 80
+   }],
+  domains: [{
+       comment: "demo",
+       name: "demo.pulumi.com",
+  }],
+  forceDestroy: true,
+});
+```
+
+```python
+import pulumi_fastly as fastly
+
+service = fastly.Servicev1("my-service",
+  backends=[{
+    "address": "127.0.0.1",
+    "name": "localhost",
+    "port": "80",
+  }],
+  domains=[{
+    "comment": "demo",
+    "name": "demo.pulumi.com",
+  }],
+  force_destroy="true",
+)
+```
+
+```go
+import (
+  fastly "github.com/pulumi/pulumi-fastly/sdk/go/fastly"
+)
+
+service, _ := fastly.NewServicev1(ctx, "test", &fastly.Servicev1Args{
+  Backends: [{
+    Address: "127.0.0.1",
+    Name: "localhost",
+    Port: 80,
+  }],
+  Domains: [{
+    Comment: "demo",
+    Name: "demo.pulumi.com"
+  }]
+  ForceDestroy: true,
+})
+```
+
+```csharp
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Pulumi;
+using Pulumi.Fastly;
+
+class Program
+{
+    static Task Main() =>
+        Deployment.Run(() => {
+            var service = new Fastly.Servicev1("test", new Fastly.Servicev1Args
```

Comment:
I haven't tried it out, so could be wrong, but just from reading it, should this be:

```suggestion
            var service = new Servicev1("test", new Servicev1Args
```

?

Reply:
You are correct - and confirmed by @mikhailshilkov